### ISSUE TITLE
Portland, OR scraper missing some valid event minutes

### ISSUE THREAD

**Issue by dphoria** [CouncilDataProject/cdp-scrapers#71]
### Describe the Bug

The last item on https://www.portland.gov/council/agenda/2021/8/18, numbered 644 seems to contain enough information including voting records for us to produce a `EventMinutesItem` from it. But we do not.

`PortlandScraper.get_matter()` fails to handle this item; returns `None`. Probably because this item does not contains "document number" in its HTML blob.

### Expected Behavior

_What did you expect to happen instead?_

Expected `get_matter()` to return `not None` for the last event minutes item on this day.

### Reproduction

_Steps to reproduce the behavior and/or a minimal example that exhibits the behavior._

```Python
from datetime import datetime
from cdp_scrapers.instances.portland import PortlandScraper
p = PortlandScraper()
events = p.get_events(begin=datetime(2021, 8, 18), end=datetime(2021, 8, 19))
print(events[0].event_minutes_items[-1].matter.name)
# 190540, corresponding to the previous item, nubmered 643
```

### Environment

_Any additional information about your environment._

-   CDP Backend Version: 3.0.0.dev22

**Comment by Shak2000** [comment]
Please assign this issue to me.

**Comment by Shak2000** [comment]
I noticed that the program the document number both in `matter.name` and in `minute_item.name`, but it does not save the number of the sections (as seen in this bug's example: 644). I am thinking of changing either `matter.name` or `minute_item.name` to the section number. Which of these keys cannot be `None`?
![image](https://user-images.githubusercontent.com/58454820/159206387-fd21fbcd-51a5-47a0-945c-9b0687938852.png)

**Comment by JacksonMaxfield** [comment]
They both are optional but it is important to know that `Matter` is for pieces of legislation so in the case you showed... I would assume you want it for both. Is there any other info we can use that won't duplicate the info? The "Motion to amend paragraph D of Exhibit A..." seems like a great description. But also I personally wouldn't worry about having the same name for a minutes item and matter. It's very common.

**Comment by Shak2000** [comment]
I will use `EventMinutesItem.minutes_item.name` as a 644 in this example. I will use leave `EventMinutesItem.matter.name` as the document number. As we see in this example, the document number is sometimes empty.

**Comment by dphoria** [comment]
I agree, let's use those item numbers like 644 for `minutes_item.name`. They seem to be always given.

For `matter.name`, I think we should do both. First use document number like we already do. If unavailable, use `minutes_item.name`. This is so lack of document number does not prevent us from creating a `matter` to store other useful information like `sponsors` and `result_status` under `matter`. What do you think?

**Comment by dphoria** [comment]
> Is there any other info we can use that won't duplicate the info? The "Motion to amend paragraph D of Exhibit A..." seems like a great description.

Agreed. The code in [`get_matter()`](https://github.com/CouncilDataProject/cdp-scrapers/blob/b2e40503e172ff0eb5a5ebdd23e32a6133685e61/cdp_scrapers/instances/portland.py#L265-L267) should ideally handle this case also, in using that text blob for `matter.title` and `minutes_item.description`.

**Comment by Shak2000** [comment]
I am looking into this issue. The problem is that minute item #644 has a different structure than the other minute items. The HTML structure we used to find the title and type does not exist for this minute item. I am thinking of whether or not it would be worth it to make the code more complicated for this specific minute item by creating a unique function to process it.

**Comment by dphoria** [comment]
Personally, I do think it would be worth the effort to be able to handle this new case. This item 644 is indeed the first example of this new structure. But it may not be the last. Plus the information itself is relatively useful/important, like `matter.title`.

The added complexity shouldn't be too bad, I think? e.g. Do what we currently do

```Python
find("div", class_="council-document__title")
```

If not found, do something like

```Python
find("div", class_="field--name-field-disposition-notes")
```

or similar.

**Comment by JacksonMaxfield** [comment]
I agree with the above plan ^^

**Comment by Shak2000** [comment]
Ok. I'll do it.
